src/styles/index.css

```
@@ -279,51 +383,52 @@
-            @apply text-gray-700 dark:text-gray-300 hover:text-teal-800 dark:hover:text-teal-300 hover:bg-gray-50 dark:hover:bg-gray-800;
+            @apply text-muted-foreground hover:text-primary hover:bg-muted/50;
           }
         }
       }
     }
   }
 
   .footer {
-    @apply bg-gradient-to-br from-gray-900 via-slate-900 to-gray-900 text-white pt-16 pb-8;
+    @apply text-white pt-16 pb-8;
+    background-color: hsl(var(--footer));
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)